src/app/container/versions/versions.component.html

```diff
@@ -101,13 +96,6 @@
           </span>
           <ng-template #noCommitID><span>{{ version.reference || 'n/a' }}</span></ng-template>
         </td>
-        <td *ngIf="tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
```

Comment:
Deleted on purpose?

Reply:
Ya, I was under the impression we're deleting the "Paths" column from both tool and workflow version table